utils/cta_dbt_helper.sh

```diff
@@ -9,41 +9,19 @@ copy_dbt_from_airbyte() {
     ALLOWED_ENVIRONMENTS="dev\nprod"
 
     ENVIRONMENT=$(echo -e $ALLOWED_ENVIRONMENTS | gum filter --placeholder "Which Airbyte Environment should be used?")
-    WORKSPACE_ID=$(gum input --placeholder "Airbyte Workspace ID ex. 814")
     
-    # Prompt if Cacher Snippet should be downloaded
-    if gum confirm "Download Cacher script to copy an Airbyte workspace?"; then
-        # Get Cacher creds from user
-        CACHER_API_KEY=$(gum input --placeholder "Enter your Cacher API Key -> https://app.cacher.io/enter?action=view_api_creds")
-        CACHER_API_TOKEN=$(gum input --placeholder "Enter your Cacher API Token -> https://app.cacher.io/enter?action=view_api_creds")
-        # This is just the ID to the Cacher snippet that holds a script to extract an Airbyte Workspace from the Airbyte server.
-        # https://docs.airbyte.com/operator-guides/transformation-and-normalization/transformations-with-dbt/#exporting-dbt-normalization-project-outside-airbyte
-        CACHER_SNIPPET_GUID="2b77280d537736f980f9" 
-
-        mkdir -p $ROOT_PATH/.cta
-        cd $ROOT_PATH/.cta
-        pipenv run python $ROOT_PATH/utils/dbt_format_utils.py getCacherScript \
-        --cacherSnippetGUID $CACHER_SNIPPET_GUID \
-        --cacherApiKey $CACHER_API_KEY \
-        --cacherApiToken $CACHER_API_TOKEN \
-        --outputPath copy_airbyte_workspace.sh
-        RET=$?
-        if [[ $RET -ne 0 ]]; then
-            echo "Failed to download Cacher Script. Exiting"
-            exit 1
-        fi  
-        chmod +x copy_airbyte_workspace.sh
-        cd $ROOT_PATH
-    fi
-
     # Run script to copy Airbyte Workspace to local
     echo "Starting job to copy Airbyte workspace.."
-    ./.cta/copy_airbyte_workspace.sh -e $ENVIRONMENT -w $WORKSPACE_ID
+    # ./copy_dbt_from_k8s_pod.sh <POD_NAME_REGEX_PATTERN> <NAMESPACE> <CONTAINER_NAME>"
+    ./copy_dbt_from_k8s_pod.sh "normalization" "airbyte" "main" 
```

Comment:
Can these paths be prepended with $ROOT_PATH. Want this script to not care which directory it is being ran from. 

```shell
./$ROOT_PATH/utils/copy_dbt_from_k8s_pod.sh "normalization" "airbyte" "main" 
mkdir -p $ROOT_PATH/.cta/airbyte_dbt_export/
mv config/* $ROOT_PATH/.cta/airbyte_dbt_export/
```